x-pack/solutions/security/plugins/security_solution/public/detection_engine/rule_creation_ui/components/status_icon/index.tsx

```diff
@@ -27,12 +26,15 @@ const RuleStatusIconStyled = styled.div`
 `;
 
 const RuleStatusIconComponent: React.FC<RuleStatusIconProps> = ({ name, type }) => {
-  const theme = useEuiTheme();
-  const color = type === 'passive' ? theme.euiColorLightestShade : theme.euiColorPrimary;
+  const { euiTheme } = useEuiTheme();
+
+  const color = type === 'passive' ? euiTheme.colors.lightestShade : euiTheme.colors.primary;
   return (
     <RuleStatusIconStyled>
       <EuiAvatar color={color} name={type === 'valid' ? '' : name} size="l" aria-label={name} />
-      {type === 'valid' ? <EuiIcon type="check" color={theme.euiColorEmptyShade} size="l" /> : null}
+      {type === 'valid' ? (
+        <EuiIcon type="check" color={euiTheme.colors.emptyShade} size="l" />
```

Comment:
Same here, `euiTheme.colors.emptyShade` is deprecated. We can use `theme.colors.backgroundBasePlain` instead.
```suggestion
        <EuiIcon type="check" color={euiTheme.colors.backgroundBasePlain} size="l" />
```